playbooks/portals-setup-following.yml

```diff
@@ -39,16 +39,28 @@
     - name: Check docker was setup
       ansible.builtin.command: docker ps
       register: docker_ps_result
-      ignore_errors: True
+      # Use failed_when: False instead of ignore_errors: True so that here isn't
+      # a red logs that are confusing to users.
+      #
+      # NOTE: failed_when: False forces docker_ps_result.failed to be False but
+      # any errors are still recording in docker_ps_result.stderr
+      failed_when: False
 
     - name: Check docker SDK for Python3 was installed
       ansible.builtin.command: python3 -m pip show docker
       register: docker_pip_result
-      ignore_errors: True
```

Comment:
Here you need to handle an error when pip or docker is not yet installed:
`failed_when: False`

Reply:
Are you saying I need `failed_when: False`? Because I have it. L57

Reply:
I have missed that, then it's ok.